src/useFirebaseUser.js

```
@@ -43,28 +43,53 @@ const defaultTokenChangedHandler = async (authUser) => {
   return response
 }
 
-const setAuthCookie = async (firebaseUser) => {
+/**
+ * Fetch the currentusers idTokenResult which contains both the idToken and the claims
+ * https://firebase.google.com/docs/reference/js/firebase.auth.IDTokenResult
+ */
+const fetchIdTokenResult = async () => {
+  try {
+    const idTokenResult = await firebase.auth().currentUser.getIdTokenResult()
+    return idTokenResult
+  } catch (e) {
+    logDebug('Error Getting Id Token Result', e)
```

I suspect we will just want to throw here instead of swallowing the error.